koku/api/common/pagination.py

```diff
@@ -212,21 +212,12 @@ def __init__(self, data_set, request, cost_type):
 
     def get_paginated_response(self, data):
         """Override pagination output."""
-        meta = self._default_meta
+        response = super().get_paginated_response(data)
+
         if self.cost_type:
-            meta["cost_type"] = self.cost_type
-        return Response(
-            {
-                "meta": meta,
-                "links": {
-                    "first": self.get_first_link(),
-                    "next": self.get_next_link(),
-                    "previous": self.get_previous_link(),
-                    "last": self.get_last_link(),
-                },
-                "data": data,
-            }
-        )
+            response.data["meta"]["cost_type"] = self.cost_type
+
+        return response
```

Comment:
This all looks great!

Let's just make this class the `ForecastListPaginator` and eliminate the `AWSForecastListPaginator`. It will be a little confusing in the future if we add another provider type and see that we are using the `AWS` paginator for all of them.